Fakebook.Posts/Fakebook.Posts.Domain/Models/Comment.cs

```diff
@@ -7,16 +7,48 @@ namespace Fakebook.Posts.Domain.Models
     public class Comment
     {
         public int Id { get; set; }
-        public string Content { get; set; }
+        private string _Content; 
```

Comment:
keep consistent with project naming convention - private fields should start with underscore and lowercase letter